README.md

```diff
@@ -99,7 +99,9 @@ To add an event to local buffer, you can call `TreasureData`'s `addEvent` or `ad
 ```
 
 
-Specify the database and table to which you want to import the events. The total length of database and table must be shorter than 129 chars.
+Specify the database and table to which you want to import the events. The total length of database and table must be shorter than 129 chars. Each table will cache no more than 10000 events.
```

Comment:
I think cache if from Keen managed storage, not specific for table within TD right ?

Reply:
We map `database.table` to collection in Keen as Keen does not have notion of database/table. So the collection is indeed map 1 - 1 to the table. Writing the doc this way helps not to introduce the Keen's collection concept in the SDK documentation.